pkg/types/ovirt/machinepool.go

```diff
@@ -39,6 +39,10 @@ type MachinePool struct {
 	// Hugepages is the size of a VM's hugepages to use in KiBs.
 	// +optional
 	Hugepages Hugepages `json:"hugepages,omitempty"`
+
+	// PreallocatedDisks sets the disks to be prepovisioned.
```

Comment:
```suggestion
	// PreallocatedDisks sets the disks to be preprovisioned.
```

But I think it sets them to be pre-allocated, not pre-provisioned.